app/api/messages/api.rb

```diff
@@ -0,0 +1,10 @@
+module Messages
+  class API < Grape::API
```

Comment:
I think we need a base API class for the whole application, maybe we can give it the same name as the app name `TempMessage::API`.

Then inside this base class we can mount a specific class for the `Messages` resource, which would be this one. What do you think? And we can also mount a class for the `Ping` endpoint which would work as a health check

Reply:
ok, so:

```ruby
module TempMessages
  class API < Grape::API
    prefix '/temp-messages'
    format :json
    formatter :json, Grape::Formatter::Jbuilder
    mount Messages::API => prefix
    mount Ping
  end
end
```